### PR TITLE
[diem-framework] spec improvements

### PR DESCRIPTION
**Pull request by Kindhearted57**
## Motivation

*DiemAccount.move*
`fun epilogue_common<Token>`
```
// The transaction sequence number is passed in to prevent any
// possibility of the account's sequence number increasing by more than
// one for any transaction.
sender_account.sequence_number = sender_account.sequence_number + 1;
```
This sequence number increasing behavior is not specified in spec function.

*DualAttestation.move*
spec for the following code is missing

```
move_to(
    dr_account,
    Limit {
        micro_xdx_limit: (initial_limit as u64)
    }
```
### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

@meng-xu-cs

### CODE REVIEW

**Review by wrwg — APPROVED** [review]

File: diem-move/diem-framework/DPN/releases/artifacts/current/build/DPNFramework/BuildInfo.yaml [hunk 1]
```diff
@@ -165,7 +165,7 @@ compiled_package_info:
     ? address: "00000000000000000000000000000001"
       name: XUS
     : DiemFramework
-  source_digest: BC083B5E2901850EFAB1BE2CCC9DA2B81D8741E89B33D4D54FB9B1BCAB0E347A
+  source_digest: DD10031D931E645385E401CEA3428946E2A4E14C1F24EFD52E0473147388FF8D
```
Comment (wrwg) [inline diem-move/diem-framework/DPN/releases/artifacts/current/build/DPNFramework/BuildInfo.yaml:168]: @tzakian fpr whether updating this causes any issues with release management.

Comment (meng-xu-cs) [reply diem-move/diem-framework/DPN/releases/artifacts/current/build/DPNFramework/BuildInfo.yaml:168]: @tzakian sorry for another ping, is it OK to update this value here?

Comment (tzakian) [reply diem-move/diem-framework/DPN/releases/artifacts/current/build/DPNFramework/BuildInfo.yaml:168]: Sorry for not seeing this earlier. This shouldn't be a problem, so no issues on my side with updating this.

Comment (meng-xu-cs) [reply diem-move/diem-framework/DPN/releases/artifacts/current/build/DPNFramework/BuildInfo.yaml:168]: Thank you!

**Review by wrwg — APPROVED** [review]

**Pull request merged**